src/java/test/org/apache/zookeeper/server/persistence/FileTxnSnapLogTest.java

```diff
@@ -113,6 +116,26 @@ public void testAutoCreateDb() throws IOException {
         attemptAutoCreateDb(dataDir, snapDir, sessions, priorAutocreateDbValue, "false", 0L);
     }
 
+    @Test
+    public void testGetTxnLogSyncElapsedTime() throws IOException {
```

Comment:
This tests the sync time, but not the bean. It is a good test case to have, but it would be also good to add server bean test for the methods added in this patch (see ZooKeeperServerBeanTest).